### PR TITLE
Mi200 counters

### PR DESCRIPTION
**Pull request by SwRaw**
Duplicate heading corrections and more improvements in MI200 performance counters.

### CODE REVIEW

**Review by Rmalavally — APPROVED** [review]
Looks good, @SwRaw

File: .wordlist.txt [hunk 1]
```diff
@@ -71,6 +71,7 @@ convolutional
 convolves
 CoRR
 CP
+CP's
```
Comment (Rmalavally) [inline .wordlist.txt:74]: CPs? @SwRaw

**Review by Rmalavally — DISMISSED** [review]
@SwRaw looks good. Thanks for your efforts.

**Comment by SwRaw** [comment]
@samjwu The spelling errors in other files are causing the spell check to fail for this PR. I am not sure who is working on those files. Let me know if I should fix errors for those files as well.

**Comment by LisaDelaney** [comment]
@SwRaw can you fix the right-aligned tables? (They should be left-aligned,  |:----|  )

**Comment by LisaDelaney** [comment]
I think we could be more consistent with capitalization throughout.

**Comment by SwRaw** [comment]
> @SwRaw can you fix the right-aligned tables? (They should be left-aligned, |:----| )

@LisaDelaney Fixed.

File: docs/conceptual/gpu-arch/mi200-performance-counters.md [hunk 2]
```diff
@@ -1,455 +1,571 @@
...
+| `SIMD`         | Single Instruction Multiple Data                                                  |
+| `sL1D`         | Scalar Level-1 Data Cache                                                         |
+| `SMEM`         | Scalar Memory                                                                     |
+| `SPI`          | Shader Processor Input                                                            |
+| `SQ`           | Sequencer                                                                         |
+| `TA`           | Texture Addressing Unit                                                           |
+| `TC`           | Texture Cache                                                                     |
+| `TCA`          | Texture Cache Arbiter                                                             |
+| `TCC`          | Texture Cache per Channel, known as L2 Cache                                      |
+| `TCIU`         | Texture Cache Interface Unit, CP's interface to memory system |
```
[the remark this answers is not in the document]
Comment (SwRaw) [reply docs/conceptual/gpu-arch/mi200-performance-counters.md]: > "CP interface-to-memory system"?

No, this won't do. It is more of "interface between CP and the memory system"

**Comment by saadrahim** [comment]
@LisaDelaney are we ready to merge this?

**Comment by LisaDelaney** [comment]
> @LisaDelaney are we ready to merge this?

Yeah, I think so...minor tweaks can be made later.

File: docs/conceptual/gpu-arch/mi200-performance-counters.md [hunk 3]
```diff
@@ -1,455 +1,571 @@
...
+| `VALUInsts` | Average number of vector ALU instructions executed per work item (affected by flow control). |
+| `VALUUtilization`  | Percentage of active vector ALU threads in a wave. A lower number can mean either more thread divergence in a wave or that the work-group size is not a multiple of 64. Value range: 0% (bad), 100% (ideal - no thread divergence). |
+| `VFetchInsts`      | Average number of vector fetch instructions from the video memory executed per work-item (affected by flow control). Excludes FLAT instructions that fetch from video memory.               |
+| `VWriteInsts`      | Average number of vector write instructions to the video memory executed per work-item (affected by flow control). Excludes FLAT instructions that write to video memory.                 |
+| `Wavefronts` | Total wavefronts. |
+| `WRITE_REQ_32B` | Total number of 32-byte effective memory writes. |
+| `WriteSize` | Total kilobytes written to the video memory. This is measured with all extra fetches and any cache or memory effects taken into account. |
+| `WriteUnitStalled` | Percentage of GPU time the write unit is stalled. Value range: 0% to 100% (bad).      |
+
+## Abbreviations
```
[the remark this answers is not in the document]
Comment (LisaDelaney) [reply docs/conceptual/gpu-arch/mi200-performance-counters.md:518]: No, I'm referring to the "meaning" colum.

**Review by samjwu — APPROVED** [review]

**Pull request merged**